src/backend/joanie/tests/payment/test_backend_lyra.py

```diff
@@ -548,9 +549,7 @@ def test_payment_backend_lyra_create_zero_click_payment(self):
         # Transaction is created
         self.assertEqual(order.invoices.first().transactions.count(), 1)
         # With 0 click payment, order state is not changed
-        old_state = order.state
-        order.refresh_from_db()
-        self.assertEqual(order.state, old_state)
+        self.assertEqual(order.state, ORDER_STATE_PENDING)
```

Comment:
Why not refreshing the order befaore checking its state ?

Reply:
I suppose that is related to the state machine: the order state is changed without need to refresh from db.

Reply:
done